src/pad/src/RDLRouter.h

```diff
@@ -166,15 +249,12 @@ class RDLRouter
     return vertex_point_map_;
   }
   const ObsTree& getObstructions() const { return obstructions_; }
-  const std::map<odb::dbNet*, std::vector<TargetPair>>& getRoutingMap() const
+  const NetRoutingTargetMap& getRoutingTargets() const
   {
-    return routing_terminals_;
-  }
-
-  const std::map<odb::dbNet*, std::vector<NetRoute>>& getRoutes() const
-  {
-    return routes_;
+    return routing_targets_;
   }
+  const std::vector<RDLRoutePtr>& getRoutes() const { return routes_; }
```

Comment:
No caller?

Do all these methods need to be public?  It seems like many are just called internally (eg getFailedRoutes).

Reply:
I believe they are there to be available to the GUI. If you have suggestions for how to structure it such that they can be private I can fix that.

Reply:
getFailedRoutes is only called from within RDLRouter.  getRoutes has no callers at all.  I haven't checked the rest.

Reply:
https://github.com/The-OpenROAD-Project/OpenROAD/blob/a68afa069d600afdc0d5037b11e8a08dff96ecfa/src/pad/src/RDLRouter.cpp#L2039
https://github.com/The-OpenROAD-Project/OpenROAD/blob/a68afa069d600afdc0d5037b11e8a08dff96ecfa/src/pad/src/RDLRouter.cpp#L1962